gradle-jdks-setup/src/main/java/com/palantir/gradle/jdks/setup/FileUtils.java

```diff
@@ -65,13 +66,13 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes _attrs) t
 
             @Override
             public FileVisitResult visitFile(Path file, BasicFileAttributes _attrs) throws IOException {
-                Files.copy(file, destination.resolve(source.relativize(file)));
+                Files.copy(file, destination.resolve(source.relativize(file)), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Do we still need to replace-existing with the copy to a temp location and then move?